test/arrays/userAPI/arrayAPItest.chpl

```diff
@@ -57,10 +57,11 @@ proc testArrayAPI2D(lbl, X: [], sliceDom, reindexDom) {
   writeln("X is:\n", X);
   writeln();
 
-  // Test read access via tuples and varargs
-  writeln("low element is: ", X.localAccess[X.domain.alignedLow]);
+  const domForLowHigh = if X.hasSingleLocalSubdomain() then X.localSubdomain() else X.domain;
+  // Test local read access via tuples and varargs
+  writeln("low element is: ", X.localAccess[domForLowHigh.alignedLow]);
```

Comment:
It seems as though it could be that, for a given array+domain, the localSubdomain could be empty or X.domain's low could not be local to the current locale.  Are you thinking that because we control the clients of this helper module, we'll just be sure not to call it in such cases?  (which is fine with me).  So, for example, with Block, all locales will have a piece and with DR we'll just be running with one locale?